registry.go

```diff
@@ -49,7 +51,8 @@ func newRegistry() *registry {
 // The match is done per packetType (iq, message, or presence) and XML tag name.
 // You can use the alias "*" as local XML name to be able to match all unknown tag name for that
 // packet type and namespace.
-func (r *registry) MapExtension(pktType packetType, name xml.Name, extension MsgExtension) {
+func (r *registry) MapExtension(pktType packetType, extension MsgExtension) {
+	name := extension.Name()
```

Comment:
Does not work alike expected yet.
We need to read the golang xml tag annotation -> a new created `xml.Name` does not contain a value in `Local` and `Space`